server/reload.go

```diff
@@ -846,6 +884,49 @@ func (s *Server) applyOptions(ctx *reloadContext, opts []option) {
 	s.Noticef("Reloaded server configuration")
 }
 
+// Update all cached debug and trace settings for every client
+func (s *Server) reloadClientTraceLevel() {
+	opts := s.getOpts()
+
+	if opts.NoLog {
+		return
+	}
+
+	update := func(c *client) {
+		// client.trace is commonly read while holding the lock
+		c.mu.Lock()
+		defer c.mu.Unlock()
```

Comment:
This is getting faster for sure but I would still avoid here with simple one liners and do explicit unlock after.

Reply:
Ok, will change.